pkg/signature/kms/azure/client_test.go

```diff
@@ -0,0 +1,90 @@
+package azure
```

Comment:
please add the license header file, check the other go files for that

Reply:
Fixed!